app/assets/stylesheets/modules/_timeline.styl

```diff
@@ -539,4 +539,11 @@ a.handout-link
   text-decoration none
   padding 5px
   &:hover, &:focus
-    border solid 1px $brand_primary
\ No newline at end of file
+    border solid 1px $brand_primary
+
+.handout-link-container
+  display flex
+  flex-wrap wrap
```

Comment:
I think this would work fine without the new div if you add the `display` and `flex-wrap` to `.block__block-content` and then add `margin-right` to `a.handout-link`.

I don't think adding those things to the block content is likely to cause any problems.

The margin-top is a little nicer than the current layout, but it mainly matters for a user who has access to the [Edit] button of the block. When that button is not there, extra space doesn't make much difference anyway, I think. And either way, it's a minor thing that I think we can leave off.

Reply:
@ragesoss I already tried that before these two solutions but .block__block-content is being used in other places also like week 1. when i add flex in .block__block-content it sets the direction to row so other things gets changed. I will attach a Screenshot later. 

Yes, We can remove margin-top

Reply:
@ragesoss I already tried that before these two solutions but .block__block-content is being used in other places also like week 1. when i add flex in .block__block-content it sets the direction to row so other things might gets changed. I will attach a Screenshot later.

Yes, We can remove margin-top